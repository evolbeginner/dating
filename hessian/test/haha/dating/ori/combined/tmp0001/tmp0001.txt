
         5         54  P

t1                              AAAAACDDEE EEEFFFFGGG GGHHIIIKKL LLLMMNNNPQ QQRRRRSSTT TVVY
t2                              AMSSSKDQEE KKTFFLYAGG STHHIIVEKF LLWMMGNNPF QSRRRRPSKT TIVY
t3                              AMASSADEEQ EKEFFFFGGG SGHHIIVKKF LLVMVGNQPL QKARRRSSTT TVVY
t4                              AMASSSDEEE EKEFFFFGGN SGHHIIVKKF LVVMLGNNPV QKRKKRSSST TVVY
t5                              AMAASKDEEE EKEFVFLGGN SGDHITTNKF LLVMMGNRPA QKKKRRASTT VVVY



    6    1    1    1    1    1    5    1    5    1    1    1    1    4    1
    1    1    3    4    1    1    1    1    1    1    1    1    1    2    1
    8    1    1    2    1    1    3    1    4    1    1    1    1    1    1
    4    1    4    1    1    1    2    2    4

