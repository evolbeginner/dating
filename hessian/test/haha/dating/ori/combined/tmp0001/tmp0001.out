AAML (in paml version 4.10.6, November 2022)  tmp0001.txt
Model: Empirical,  (/mnt/hd1/home/sishuo/software/phylo/paml/latest/dat/lg.dat) dGamma (ncatG=4) ns =   5  ls = 100


Frequencies..
                                    A      R      N      D      C      Q      E      G      H      I      L      K      M      F      P      S      T      W      Y      V
t1                             0.10000 0.07000 0.05000 0.06000 0.01000 0.03000 0.09000 0.10000 0.02000 0.03000 0.11000 0.03000 0.03000 0.07000 0.04000 0.05000 0.03000 0.00000 0.04000 0.04000
t2                             0.09000 0.07000 0.04000 0.05000 0.00000 0.02000 0.07000 0.06000 0.02000 0.04000 0.10000 0.06000 0.04000 0.07000 0.05000 0.09000 0.04000 0.01000 0.05000 0.03000
t3                             0.09000 0.06000 0.03000 0.05000 0.00000 0.03000 0.08000 0.10000 0.02000 0.02000 0.10000 0.05000 0.03000 0.08000 0.04000 0.08000 0.03000 0.00000 0.04000 0.07000
t4                             0.07000 0.05000 0.05000 0.05000 0.00000 0.01000 0.09000 0.09000 0.02000 0.02000 0.09000 0.07000 0.03000 0.08000 0.04000 0.10000 0.02000 0.00000 0.04000 0.08000
t5                             0.10000 0.06000 0.05000 0.06000 0.00000 0.01000 0.09000 0.09000 0.01000 0.01000 0.10000 0.07000 0.04000 0.06000 0.04000 0.06000 0.04000 0.00000 0.04000 0.07000

Homogeneity statistic: X2 = 0.25989 G = 0.25359 

Average                        0.09000 0.06200 0.04400 0.05400 0.00200 0.02000 0.08400 0.08800 0.01800 0.02400 0.10000 0.05600 0.03400 0.07200 0.04200 0.07600 0.03200 0.00200 0.04200 0.05800

# constant sites:     61 (61.00%)
ln Lmax (unconstrained) = -371.628634

AA distances (raw proportions of different sites)

t1             
t2               0.2700
t3               0.1700  0.2300
t4               0.1900  0.2400  0.1100
t5               0.2300  0.2800  0.1700  0.1700

TREE #  1:  (2, (3, (4, 5)), 1);   MP score: 56
lnL(ntime:  7  np:  8):   -573.810670      +0.000000
   6..2     6..7     7..3     7..8     8..4     8..5     6..1  
 0.257540 0.000000 0.055459 0.031420 0.049371 0.164451 0.157755 0.835307
SEs for parameters:
 0.070177 0.114076 0.029549 0.022514 0.028531 0.054888 0.055241 0.411911

tree length =  0.715997

(2: 0.257540, (3: 0.055459, (4: 0.049371, 5: 0.164451): 0.031420): 0.000000, 1: 0.157755);

(t2: 0.257540, (t3: 0.055459, (t4: 0.049371, t5: 0.164451): 0.031420): 0.000000, t1: 0.157755);

Detailed output identifying parameters

alpha (gamma, K = 4) =  0.83531
rate:   0.10305  0.42097  0.96628  2.50971
freq:   0.25000  0.25000  0.25000  0.25000

Time used:  0:00
